docs/concepts/09_glossary.md

```diff
@@ -0,0 +1,157 @@
+
+DID authentication is the process of proving that a DID owner has control over a DID, typically by using a digital signature or other cryptographic mechanism.
+
+### DID Communication
+
+DID communication refers to the use of DIDs to enable secure and decentralized communication between two or more parties, without the need for a central intermediary.
+
+### DID Controller
+
+A DID controller is an entity that has control over a DID, such as the entity that created the DID or an entity that has been authorized by the DID owner to manage the DID.
+
+### DID Document
+
+A DID document is a JSON-LD document that contains information about a DID, including public keys, service endpoints, and other metadata.
+
+### DID Method
+
+A DID method is a set of rules and specifications for how a DID is created, resolved, and managed on a particular network or platform.
+
+### DID Resolver
+
+A DID resolver is a software component that can resolve a DID to a DID Document, which contains information about the DID, such as public keys, service endpoints, and other metadata.
+
+### Self-Sovereign Identity
+
+A decentralized digital identity management system that enables individuals to own and control their identity information using secure digital technologies, such as blockchain.
+SSI eliminates the need for intermediaries and provides individuals with greater privacy, security, and control over their personal data.
+It is an emerging concept that has the potential to transform how identity is managed and verified across various sectors.
+
+### Verifiable Credentials
+
+Verifiable credentials are digital credentials that can be used to prove claims about a person, organization, or thing, and are designed to be portable, interoperable, and privacy-preserving.
+Verifiable credentials are often associated with DIDs and can be stored and managed using a DID-based identity system.
+
+## KILT Protocol Specific Terms
+
+### Attestation
+
+The act of providing evidence or confirming the validity of a claim, typically performed by a trusted attester.
+
+### Attester
+
+A person, organization or entity that verifies and validates the identity or credentials of an individual in a digital or physical setting.
+Attesters play a critical role in building trust and establishing the authenticity of information and data shared between parties.
+
+### Claimer
+
+An individual or entity that asserts a claim or statement about their identity or qualifications.
+The claimer can use digital credentials, such as badges or certificates, to provide evidence of their claims, which can be verified by third-party entities or systems.
+
+### Claim Type (CType)
+
+A specific type of claim that can be made about an individual, such as their education, work experience, or identity information.
+Each claim type has a defined set of attributes that must be provided to support the claim, and it can be used to generate verifiable credentials that can be shared with others.
```

Comment:
Yes, it can support data integrity by creating a structure to check the data types against. 

```suggestion
Each claim type has a defined set of attributes that must be provided to support the claims data type and structure. It can be used to generate verifiable credentials that can be shared with others. [CTypes concept page for more details](https://docs.kilt.io/docs/concepts/credentials/ctypes)
```